build.gradle

```diff
@@ -360,19 +360,6 @@ subprojects {
   test.dependsOn('checkstyleMain', 'checkstyleTest')
 
   if (!JavaVersion.current().isJava9Compatible()) {
-    findbugs {
-      toolVersion = "3.0.1"
-      excludeFilter = file("$rootDir/gradle/findbugs-exclude.xml")
-      ignoreFailures = false
-    }
-    test.dependsOn('findbugsMain')
-
-    tasks.withType(FindBugs) {
-      reports {
-        xml.enabled(project.hasProperty('xmlFindBugsReport'))
-        html.enabled(!project.hasProperty('xmlFindBugsReport'))
-      }
-    }
```

Comment:
I assume this is not intentional :)

Reply:
@ijuma I made this to skip findbug and speedup test execution while I am still debugging my test :)

Reply:
@ijuma The code is almost ready. I will let you know once it is done today.